src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -391,53 +401,57 @@ std::optional<units::length::inch_t> DragonLimelight::EstimateTargetXDistance()
     std::optional<int> aprilTagID = GetAprilTagID();
     if (!aprilTagID)
```

Comment:
this code needs to check for aprilTagId being null or it'll throw a bad optional access exception.  std::optional<int> aprilTagID{GetAprilTagID()};
    if (!aprilTagID.has_value() && targetPitch.has_value())
    {